src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello, world!</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <p>Hello, world!</p>
+  </body>
+
+</html>
```

Comment:
- Leave one empty line at the end of the file
https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/